sources/packages/backend/apps/api/src/testHelpers/auth/_tests_/e2e/aest-token-helpers.getAESTToken.e2e-spec.ts

```diff
@@ -0,0 +1,18 @@
+import { JwtService } from "@nestjs/jwt";
+import { IUserToken, Role } from "../../../../auth";
+import { AESTGroups, getAESTToken } from "../../..";
+
+const jwtService = new JwtService();
+
+describe("(e2e)-getAESTToken()", () => {
```

Comment:
I would recommend using the "Auth" as a prefix, like `Auth(e2e)-getAESTToken()`.